pilot/pkg/features/pilot.go

```diff
@@ -424,6 +424,9 @@ var (
 	DeltaXds = env.RegisterBoolVar("ISTIO_DELTA_XDS", false,
 		"If enabled, pilot will only send the delta configs as opposed to the state of the world on a "+
 			"Resource Request")
+
+	SharedMeshConfig = env.RegisterStringVar("SHARED_MESH_CONFIG", "",
+		"Additional config map to load for shared MeshConfig settings").Get()
```

Comment:
nit: would be nice to comment here if it takes precedence over the standard mesh config or not